modules/kuberay-cluster/main.tf

```diff
@@ -44,7 +44,8 @@ resource "helm_release" "ray-cluster" {
       security_context                  = local.security_context
       secret_name                       = var.db_secret_name
       cloudsql_instance_connection_name = local.cloudsql_instance_connection_name
-      image_tag                         = var.enable_gpu ? "2.9.3-py310-gpu" : "2.9.3-py310"
+      image                             = var.use_custom_image ? "us-central1-docker.pkg.dev/ai-on-gke/rag-on-gke/ray-image" : "rayproject/ray"
+      image_tag                         = var.enable_gpu ? "2.9.3-py310-gpu" : var.use_custom_image ? "2.9.3-py310-gpu" : "2.9.3-py310"
```

Comment:
@chiayi can we simplify this by also publishing 2.9.3-py310 to artifact registry? Even withput GPUs I think this would benefit

Reply:
Yes, I will work on creating an image for 2.9.3-py310 as well.